prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -646,7 +646,23 @@ void rxr_pkt_handle_receipt_recv(struct rxr_ep *ep,
 		return;
 	}
 
-	rxr_cq_handle_tx_completion(ep, tx_entry);
+	if (!(tx_entry->rxr_flags & RXR_LONG_MSG)) {
```

Comment:
Not sure why we need ```RXR_LONG_MSG```?  Only DC protocol will get receipt. We can use tx_entry->bytes_acked to distinguish DC_LONG and DC_MEDIUM/DC_EAGER. Right?

Reply:
`RXR_LONG_MSG ` flag is removed, resolving.